README.md

```diff
@@ -26,7 +26,14 @@ or multiple solutions
 
 `dotnet consolidate -s YourSolution.sln AnotherSolution.sln`
 
-If the tool finds discrepancies between projects, it exits with non-success status code and prints these discrepancies.
+You can also optionally specify the a package id if you want only a single package to be consolidated
```

Comment:
Let's use the `id` word in uppercase.
Example:
`a package ID`
`a list of package IDs`